sympy/core/sympify.py

```diff
@@ -273,6 +273,40 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     >>> kernS(s)
     -2*(-(-x + 1/x)/(x*(x - 1/x)**2) - 1/(x*(x - 1/x))) - 1
 
+    Parameters
+    ==========
+
+    a :
+        - any object defined in SymPy
+        - standard numeric python types: int, long, float, Decimal
+        - strings (like "0.09" or "2e-19")
```

Comment:
I think that this should better be exampled with symbolic expressions like how `"sin(x)"` is comprehended.
And the string input also cooperates with other keys.

Reply:
For convert_xor and rational, the input has to be a string. Even for evaluate the input has to be string for to give proper results. Should that be mentioned too?
![l](https://user-images.githubusercontent.com/51290447/85364185-36ca8d80-b540-11ea-85b0-a711a2945aa8.png)

Reply:
Yes, but you should note that they come from how python arithmetics are done in place, even before passing to sympify

Reply:
Yes, should I mention that?

Reply:
Yes, because it's quite a common gotcha that people do.

Reply:
Done